numpy/core/_add_newdocs.py

```diff
@@ -4503,18 +4503,18 @@
     avoided on arrays defined by slices, transposes, fortran-ordering, etc.:
 
     >>> x = np.array([[1,2,3],[4,5,6]], dtype=np.int16)
-    >>> y = x[:, 0:2]
+    >>> y = x[:, ::2]
     >>> y
-    array([[1, 2],
-           [4, 5]], dtype=int16)
+    array([[1, 3],
+           [4, 6]], dtype=int16)
     >>> y.view(dtype=[('width', np.int16), ('length', np.int16)])
     Traceback (most recent call last):
         ...
     ValueError: To change to a dtype of a different size, the array must be C-contiguous
     >>> z = y.copy()
     >>> z.view(dtype=[('width', np.int16), ('length', np.int16)])
-    array([[(1, 2)],
-           [(4, 5)]], dtype=[('width', '<i2'), ('length', '<i2')])
+    array([[(1, 3)],
+           [(4, 6)]], dtype=[('width', '<i2'), ('length', '<i2')])
```

Comment:
Not the fault of this pr, but will this be true on big-endian ('>') machines?

Reply:
Probably not. The numbers will be the same if course, but it might be worth changing `int16` to `'<i2'` just to be sure.

Reply:
This is actually an interesting inconsistency. If I change the original type of `x` to `'<i2'`, the output from the view will be correct, but `y` will be incorrect on big-endian systems. On the other hand if I leave it as-is, `y` will be correct but the view will be incorrect. The fix would be to display dtypes more consistently (either both cases `int16` or both `<i2`) rather than modifying the doctest. I will leave it as-is since the system currently running the doctests seems to be little-endian.